UtopiaCityTest/Controllers/HousingSystem/HousingSystemControllerTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿//using AutoMapper;
+//using Microsoft.AspNetCore.Mvc;
+//using Microsoft.EntityFrameworkCore;
+//using Moq;
+//using System;
+//using System.Collections.Generic;
```

Comment:
?

Reply:
This controller test has some issues which I am planning ressolve later